.github/workflows/post_to_mastodon.yml

```
@@ -16,12 +16,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Post to Mastodon
+        uses: actions/checkout@v4
         uses: nick-fields/retry@v2
```

actionlint (1.7.4)

key "uses" is duplicated in element of "steps" section. previously defined at line:19,col:9

(syntax-check)